### PR TITLE
Menu hamburger fold

### PR DESCRIPTION
**Pull request by yvesdaxmaz**
Convert the header navigation bar to an hamburger when screen size is less than 770px

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "_includes"]
 	path = _includes
-	url = https://github.com/jmapio/jmap.git
+	url = https://github.com/yvesdaxmaz/jmap.git
```
Comment (neilj) [inline .gitmodules]: This must not be changed.

File: _layouts/default.html [hunk 2]
```diff
@@ -18,6 +18,12 @@
     <li class="nav-item"><a class="nav-link" href="software.html">Get it</a></li>
     <li class="nav-item"><a class="nav-link" href="https://proxy.jmap.io">Try It</a></li>
 </ul>
+<div id="humberger-menu">
+    <input type="checkbox" />
+    <span></span>
+    <span></span>
+    <span></span>
+</div>
```
Comment (neilj) [inline _layouts/default.html]: Since the elements are not used without JavaScript, they should be added by the JavaScript.

File: toc.js [hunk 3]
```diff
@@ -139,3 +139,18 @@ window.addEventListener( 'scroll', function () {
...
+
+    $checkbox.addEventListener('change', function(event) {
+        var checkbox = event.target;
+        if(checkbox.checked) {
+            $menu.classList.add('activate');
+        } else {
+            $menu.classList.remove('activate');
+        }
+    });
+})
```
Comment (neilj) [inline toc.js]: This is not generating the table of contents, so should not be in `toc.js`; it should be its own script. The code style should match the code style used elsewhere.

**Review by neilj — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (yvesdaxmaz) [reply .gitmodules]: I did that to have hand on both jmapio.github.io project and jmap.io on within and same directory one as a parent and another as a submodule. Like later on I could have to made some changes on the jmap.io repository

File: _layouts/default.html (hunk 2 above)
Comment (yvesdaxmaz) [reply _layouts/default.html]: Done

File: toc.js (hunk 3 above)
Comment (yvesdaxmaz) [reply toc.js]: Done

File: toc.js [hunk 4]
```diff
@@ -138,4 +138,4 @@ window.addEventListener( 'scroll', function () {
     }
 }, false );
 
-}( document, window ) );
+}( document, window ) );
```
Comment (neilj) [inline toc.js:141]: Please remove this redundant change.

File: hamburger_menu.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+document.addEventListener('DOMContentLoaded', function() {
+
+    var menu = document.getElementById('nav');
+    var hamburger_menu = document.createElement('div');
+    hamburger_menu.id = "humberger-menu";
```
Comment (neilj) [inline hamburger_menu.js]: Please use consistent spelling; I suggest `hamburger` rather than `humberger`.

**Review by neilj — CHANGES_REQUESTED** [review]
Please also remove the change to the submodule – this should not be touched.

**Pull request closed without merging**